benchmark/buffers/buffer-indexof.js

```diff
@@ -15,7 +16,8 @@ var bench = common.createBenchmark(main, {
 
 function main(conf) {
   var iter = (conf.iter) * 100000;
-  var aliceBuffer = fs.readFileSync(__dirname + '/../fixtures/alice.html');
+  var aliceBuffer = fs.readFileSync(
+    path.resolve(__dirname, '../fixtures/alice.html'));
```

Comment:
Wouldn't having the last parenthesis on the next line be more consistent (and IMHO better looking) with the changes made in `benchmark/compare.js`?